### ISSUE TITLE
[BUG] Not pulling all historical data

### ISSUE THREAD

**Issue by RogerK888** [diegomanuel/binance-to-google-sheets#54]
**Describe the bug**
Some pairs have a buy and sell transaction on the Binance server, and these can be seen on the order history page.

I noticed for example your scripts pull COTI/USDT 'BUY" data, but not the SELL data, even though they were completed the same day.

**To Reproduce**
When I did a comparison of Bought vs Sold data in the "Done Orders" and "Orders Complete" tables generated by this script, I noticed there were no SELL records for the pair COTI/USDT. "BUY" records were present.

I went to my Binance account directly, and checked to see if those records were in the historical data online, and they were. 

Unable to determine why those records were missing in this script via the code.

**Expected behavior**
Pull all data. 

**Screenshots**
N/A
**Additional context**
N/A

**Comment by niesteszeck** [comment]
I have a similar issue, but with AVAX/USDT. If I check the history on binance web it show me 2 records (one buy, one sell), but on the Orders Table sheet, there isn't any record for AVAX

**Comment by nickpb** [comment]
same here. Orders Table not updating the new records

**Comment by niesteszeck** [comment]
Hi there. I have checked the execution history on appscript dashboard and I see a lot of errors and retries. Looking at them I realice that I have a huge list of pairs to retrieve, so I divide it in a lot of sheets with a maximum of 5 pairs to retrieve between a sheet, for example:

- sheet 1 formula: `=BINANCER("orders/table", 'Dashboard USDT'!$A10:$A15,"ticker: USDT")`
- sheet 2 formula: `=BINANCER("orders/table", 'Dashboard USDT'!$A16:$A20,"ticker: USDT")`
- and so on

Also I started the first sheet and after is was full filled, containing all the historical data, I create the second sheet whit the new range. I'm repeating this process till I have all my history of all my pair .

Is a slow process and in the readme is detailed that is unknown the behaviour of a big list of pairs, so I'm following the advises there, using at most 5 pairs peer formula in a sheet, having patient waiting it to be filled, adding a new sheet with 5 new pairs (my advise), and is working like a charm so far
